### PR TITLE
Exposing root as homedir

### PR DESCRIPTION
**Pull request by mateusz-garlacz**
This fixes exposing slash for home directory. Redirecting to user directory will come next. This PR also contains #127 .

### CODE REVIEW

File: jupyter-config/jupyter_cs3_config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"reva_host": "127.0.0.1:19000",
 	"auth_token_validity": "3600",
 	"endpoint": "/",
-	"home_dir": "/home",
```
Comment (diocas) [inline jupyter-config/jupyter_cs3_config.json:6]: I wouldn't change the default, unless you think the shares will always need access to the `myshares` folder.. From other changes bellow, this seems to be what you're doing, is that correct?

File: src/infobox.tsx [hunk 2]
```diff
@@ -211,17 +211,10 @@ const Shares = (props: ShareProps): JSX.Element => {
...
-      });
-
-      const resource =
-        home.path +
-        '/' +
-        props.fileInfo.path
-          .replace('cs3driveShareByMe:', '')
-          .replace('cs3drive:', '')
-          .replace('cs3driveShareWithMe:', '');
+      const resource = props.fileInfo.path
```
Comment (diocas) [inline src/infobox.tsx:214]: Why isn't this info removed from the backend instead? If I understand correctly, you already assume the drives are not exposed to the frontend, no?

**Review by diocas — COMMENTED** [review]
Please rebase

File: jupyter-config/jupyter_cs3_config.json (hunk 1 above)
Comment (mateusz-garlacz) [reply jupyter-config/jupyter_cs3_config.json]: done

File: src/infobox.tsx (hunk 2 above)
Comment (mateusz-garlacz) [reply src/infobox.tsx:214]: After all there are no changes in this file. This is a matter of formatting, the replacing above is in master for months.

Comment (mateusz-garlacz) [reply src/infobox.tsx:214]: After all there are no changes in this file. This is a matter of formatting, the replacing above is in master for months.

**Comment by dagl** [comment]
@diocas please take a look

File: src/infobox.tsx (hunk 2 above)
Comment (diocas) [reply src/infobox.tsx:214]: I understand it was there, it was necessary, I just wonder with the changes done in the backend if this was address or not. But it's fine to keep it for a bit longer.

Comment (diocas) [reply src/infobox.tsx:214]: @mateusz-garlacz please let me know it it's necessary, otherwise let's merge.

Comment (mateusz-garlacz) [reply src/infobox.tsx:214]: @diocas it is still necessary to keep it

**Review by diocas — APPROVED** [review]

**Comment by diocas** [comment]
@dagl @mateusz-garlacz was this tested? What config do I need to put? I've set `"home_dir": "/"` and when I try to navigate inside a folder, I get the message `Directory not found: "cs3drive:"` and the log message `[handlers:617] /eos is not a file`

**Comment by diocas** [comment]
It was merged but I merged based on the changes seen, I didn't test it before. The problems are the ones I mentioned, I cannot navigate to folders from the root... Please check and open a PR with a fix.

**Comment by dagl** [comment]
> 

@diocas We just checked and it works correctly for both me and @mateusz-garlacz on user ailldev. Maybe try building the plugin on a fresh virtual env?

**Pull request merged**